judge/fixtures/language_small.json

```diff
@@ -1,122 +1,274 @@
+      "pk":2,
+      "fields":{
+         "key":"GAS64",
+         "name":"Assembly (x64)",
+         "short_name":null,
+         "common_name":"Assembly",
+         "ace":"assembly_x86",
+         "pygments":"gas",
+         "template":"",
+         "info":"",
+         "description":"",
+         "extension":""
+      }
+   },
+   {
+      "model":"judge.language",
+      "pk":3,
+      "fields":{
+         "key":"AWK",
+         "name":"AWK",
+         "short_name":"AWK",
+         "common_name":"Awk",
+         "ace":"AWK",
+         "pygments":"awk",
+         "template":"",
+         "info":"",
+         "description":"",
+         "extension":""
+      }
+   },
+   {
+      "model":"judge.language",
+      "pk":4,
+      "fields":{
+         "key":"C",
+         "name":"C",
+         "short_name":null,
+         "common_name":"C",
+         "ace":"c_cpp",
+         "pygments":"c",
+         "template":"",
+         "info":"",
+         "description":"Compile options: `gcc -std=c99 -Wall -O2  -fmax-errors=5 -lm -march=native -s`",
+         "extension":""
+      }
+   },
+   {
+      "model":"judge.language",
+      "pk":5,
+      "fields":{
+         "key":"CPP03",
+         "name":"C++03",
+         "short_name":"C++03",
+         "common_name":"C++",
+         "ace":"c_cpp",
+         "pygments":"cpp",
+         "template":"",
+         "info":"",
+         "description":"Compile options: `g++ -Wall -O2 -fmax-errors=5 -lm -march=native -s`",
+         "extension":""
+      }
+   },
+   {
+      "model":"judge.language",
+      "pk":6,
+      "fields":{
+         "key":"CPP11",
+         "name":"C++11",
+         "short_name":"C++11",
+         "common_name":"C++",
+         "ace":"c_cpp",
+         "pygments":"cpp",
+         "template":"",
+         "info":"",
+         "description":"Compile options: `g++ -std=c++11 -Wall -O2 -fmax-errors=5 -lm -march=native -s`",
+         "extension":""
+      }
+   },
+   {
+      "model":"judge.language",
+      "pk":7,
+      "fields":{
+         "key":"PERL",
+         "name":"Perl",
+         "short_name":null,
+         "common_name":"Perl",
+         "ace":"perl",
+         "pygments":"perl",
+         "template":"",
+         "info":"",
+         "description":"",
+         "extension":""
+      }
+   },
+   {
+      "model":"judge.language",
+      "pk":8,
+      "fields":{
+         "key":"PY3",
+         "name":"Python 3",
+         "short_name":null,
+         "common_name":"Python",
+         "ace":"python",
+         "pygments":"python3",
+         "template":"",
+         "info":"",
+         "description":"",
+         "extension":""
+      }
+   },
+   {
+      "model":"judge.language",
+      "pk":9,
+      "fields":{
+         "key":"JAVA8",
+         "name":"Java 8",
+         "short_name":null,
+         "common_name":"Java",
+         "ace":"java",
+         "pygments":"java",
+         "template":"",
+         "info":"",
+         "description":"",
```

Comment:
Since you have them for C/C++, runtime flags would be nice for everyone's favourite runtime too.